topics/tutorials/custom_language_support/structure_aware_navbar.md

```diff
@@ -0,0 +1,37 @@
+[//]: # "title: 15. Structure Aware Navigation Bar"
```

Comment:
the syntax for title must be fixed, see https://plugins.jetbrains.com/docs/intellij/sdk-style.html#page-format

Reply:
Fixed.